### DISCUSSION TITLE
Re-establish a function for short tap on compass sign

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#381] [Q&A] [answered]
The compass sign does not respond to a short tap for a long time, i.e. the function is orphaned. 
There were some suggestions in the past for using this function, which I would like to list here. 

The screenshot tool is currently integrated in "Tools" and is therefore only of limited use (see https://github.com/devemux86/cruiser/discussions/95#discussioncomment-5410596). 
There was already a suggestion to use the compass sign short tap for it (see https://github.com/devemux86/cruiser/discussions/57#discussioncomment-5101711). It would be nice if this could be realized, even temporarily. 
Because: It would be even more interesting to be able to place a freely selectable function for the short tap on the compass symbol (see https://github.com/devemux86/cruiser/discussions/106#discussioncomment-5469803), but this would certainly be more complex to implement. 

Btw: in the compass sign long tap selection, I would still like to add the dynamic display of the remaining route (see https://github.com/devemux86/cruiser/discussions/217#discussioncomment-6583910) - as in Kurviger 2.2.28 (my K2 subscription ends on 12.12.). In my opinion, this is an interesting alternative when navigating in the car. 

Thanks a lot Emux for considering ...

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for searching all discussions!

> place a freely selectable function

Custom action on compass (or other button) is a very interesting idea.

Another option for it could be to quickly add favorites:
- https://github.com/devemux86/cruiser/discussions/323

> dynamic display of the remaining route

The route overview (on any button) is also discussed here:
- https://github.com/devemux86/cruiser/discussions/94

**Comment by devemux86** [comment, marked as answer]
Let's discuss possible use of the tap on compass in:
- https://github.com/devemux86/cruiser/discussions/405
